internal/database/mariadb/init/schema.sql

```diff
@@ -12,8 +12,10 @@ create table if not exists Component
     component_name       varchar(256)                          not null,
     component_type       varchar(256)                          not null,
     component_created_at timestamp default current_timestamp() not null,
+    component_created_by varchar(256)                          null,
```

Comment:
Oh, well this needs to be a foreign key to a User I guess!

@MR2011  @dorneanu  WDYT?

Reply:
I still think `User` is misleading here. A component can be as well created by a scanner, right?